dataset/aoi.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, raster: Union[Path, str],
             If True, no rasterio.DatasetReader will be generated in __init__. User will have to call read raster later.
```

Comment:
looks the docstring paramenert need to be adjusted:
@param write_multiband -> @param write_dest_raster